packages/grpcweb-transport/src/grpc-web-transport.ts

```diff
@@ -70,12 +70,22 @@ export class GrpcWebFetchTransport implements RpcTransport {
     }
 
 
-    clientStreaming<I extends object, O extends object>(/*method: MethodInfo<I, O>, options: RpcOptions*/): ClientStreamingCall<I, O> {
-        throw new RpcError('Client streaming is not supported by grpc-web', GrpcStatusCode[GrpcStatusCode.UNIMPLEMENTED]);
+    clientStreaming<I extends object, O extends object>(method?: MethodInfo<I, O>/*, options: RpcOptions*/): ClientStreamingCall<I, O> {
+        const e = new RpcError('Client streaming is not supported by grpc-web', GrpcStatusCode[GrpcStatusCode.UNIMPLEMENTED]);
+        if (method) {
+            e.methodName = method.name;
+            e.serviceName  = method.service.typeName;
+        }
+        throw e;
```

Comment:
We are safe to make the argument required. `interface RpcTransport` requires both arguments - I think I just commented them out because my IDE complained about them being unused. 

```suggestion
    clientStreaming<I extends object, O extends object>(method: MethodInfo<I, O>/*, options: RpcOptions*/): ClientStreamingCall<I, O> {
        const e = new RpcError('Client streaming is not supported by grpc-web', GrpcStatusCode[GrpcStatusCode.UNIMPLEMENTED]);
        e.methodName = method.name;
        e.serviceName  = method.service.typeName;
        throw e;
```

Reply:
I made this optional so that the previous unit tests would pass (it didn't pass in the `MethodInfo`). Making this optional was a smaller change, but I will make this required and update the unit test.

Reply:
Ah, I was unaware. Thanks for the update!